code/modules/mob/living/simple_animal/bot/ed209bot.dm

```diff
@@ -90,32 +89,34 @@
 	. = ..()
 	if(. & EMP_PROTECT_SELF)
 		return
-	if (severity >= 2)
-		new /obj/effect/temp_visual/emp(loc)
-		var/list/mob/living/carbon/targets = new
-		for(var/mob/living/carbon/C in view(12,src))
-			if(C.stat==DEAD)
-				continue
-			targets += C
+	if(severity <= 1)
+		return
+	new /obj/effect/temp_visual/emp(loc)
+	var/list/mob/living/carbon/targets = new
+	for(var/mob/living/carbon/C in view(12,src))
+		if(C.stat == DEAD)
+			continue
+		targets += C
+	if(!targets.len)
+		return
+	if(prob(50))
+		var/mob/toshoot = pick(targets)
+		if(toshoot)
+			targets -= toshoot
+			if(prob(50) && emagged < 2)
```

Comment:
`emagged` being set to numbers like 2 and whatnot is dumb, within scope to fix all that to *at least* defines?

Reply:
I removed emagged going up to 2 and its just TRUE/FALSE now. I can move it to a separate PR if wanted but I didn't see reason to keep it around if even Cyborg covers can be closed once emagged open.